### DISCUSSION TITLE
jmap stop sending email

### DISCUSSION THREAD

**Discussion by mjac1521** [stalwartlabs/jmap-server#48] [Q&A]
suddenly the jmap server stop sending emails. I added debug as recommended to see if something popup I even use trace to get more verbose but no luck. the server is not even able to send a email locally to the same account, every email after hit send go straight to outbox and stay there forever. :(

Sincerely Maykel

**Comment by mdecimus** [comment]
Hi,

Please post here your JMAP configuration file or the command line parameters you are using. Also, if you are using Stalwart SMTP please also post the SMTP config file.

**Comment by mjac1521** [comment]
Please see the jmap and smtp config files here bellow. I replaced my domain for "example". When I installed jmap and smtp I made small changes like domain, certificate using letsencrypt, and conecting the smtp to jmap using lmtp and the submition from jmap to the smtp, both services jmap and smtp are in the same server. 
  
[smtp config.txt](https://github.com/stalwartlabs/jmap-server/files/11543144/smtp.config.txt)
[jmap config.txt](https://github.com/stalwartlabs/jmap-server/files/11543146/jmap.config.txt)

Sincerely Maykel

**Comment by mdecimus** [comment]
I think the problem is that the SMTP server is rejecting the connection because the JMAP server is not authorized to relay messages through it. There are two possible solutions, (1) configure the JMAP server to authenticate with the SMTP server or (2) enable relay for all connections coming from localhost which could be done as follows:

```
relay = [ { if = "authenticated-as", ne = "", then = true }, 
              { if = "remote-ip", eq = "127.0.0.1", then = true }, 
              { else = false } ]
```

In any case, the exact reason why the SMTP connection is failed should be available on the JMAP logs. If you are using syslog, they should be under /var/log/messages or similar.

**Reply by mjac1521** [reply]
Thanks a lot mdecimus that solve the issue.  🥇
